descriptors/ModuleDescriptor-template.json

```diff
@@ -998,6 +998,19 @@
         }
       ]
     },
+    {
+      "id": "orders.routing-list",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/orders/routing-lists/{id}/template",
+          "permissionsRequired": [
+            "orders.routing-list-template.item.get"
+          ]
```

Comment:
we should add module permission to be able to call mod-template-engine requests from non-admin user

Reply:
thanks, done

Reply:
forgot to mention, we also need to add module permission to get routing list from storage and to get users